test/general/logging_test.c

```diff
@@ -0,0 +1,20 @@
+#include "logging_macros.h"
+#include <unistd.h>
+
+#define TOTAL_LOOP_CASES 10
+
+/**
+* @brief test for testing LF_TIMESTAMP_PRINT_DEBUG macro
+* must be in LOG_LEVEL LOG_LEVEL_DEBUG
+*/
+int main()
+{
+    LF_PRINT_DEBUG("Start time: %ld", lf_time_start());
+    for(int i; i<TOTAL_LOOP_CASES; i++)
+    {
+        LF_TIMESTAMP_PRINT_DEBUG("[Test case: %d]testing timed debug prints", i);
```

Comment:
How are you going to make sure that this is printed correctly? You will need to add some error-checking code which will fail if your macro does not work as intended. You can do this either using `lf_print_error_and_exit` or `assert`. Please see following examples.
- `lf_print_error_and_exit`: See the following examples:
  - https://github.com/lf-lang/reactor-c/blob/main/test/scheduling/scheduling_api_test.c#L19
  - https://github.com/lf-lang/reactor-c/blob/main/test/general/utils/hashmap_test.c#L45
- `assert`: See the following example:
  - https://github.com/lf-lang/reactor-c/blob/main/test/general/utils/hashset_test.c#L24